BSP/Simulator/Src/BSP_Timer.c

```diff
@@ -36,9 +36,9 @@ int update_Time(int row)
 
 }
 
-int writeReload(int reload)
+int8_t BSP_Timer_Init(int8_t reload)
 {
-  FILE *CANdata = fopen("CAN.csv","w");
-  fprintf(CANdata,"0x%x,%d",reload,reload);
-  fclose(CANdata);
+  FILE *TIMERdata = fopen("Timer.csv","w");
+  fprintf(TIMERdata,"0x%x",reload);
```

Comment:
When writing to the file, it should write 2 numbers (the CURRENT value and the RELOAD value)